shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/oracle/ddl/OracleAlterSynonymStatement.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.sql.parser.sql.dialect.statement.oracle.ddl;
+
+import lombok.ToString;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.AbstractSQLStatement;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.ddl.DDLStatement;
+import org.apache.shardingsphere.sql.parser.sql.dialect.statement.oracle.OracleStatement;
+
+/**
+ * Oracle alter synonym statement.
+ */
+@ToString
+public final class OracleAlterSynonymStatement extends AbstractSQLStatement implements DDLStatement, OracleStatement {
```

Comment:
@tristaZero Hi, do you think the OracleAlterSynonymStatement definition is reasonable?

Reply:
Hi @Liangda-w , @wgy8283335 
Is there any members in this class?

Reply:
Hi @tristaZero,

what do you mean by members? 
Since ALTER SYNONYM is Oracle specific, I didn't create the common `AlterTableStatement` class.

Reply:
There is not any member in this class. @tristaZero

Reply:
Got it, so this PR is the right time to merge?

Reply:
CI still failure. @tristaZero @Liangda-w